frontend/app/routes/_gcweb-app.personal-information.home-address.edit.tsx

```diff
@@ -87,15 +88,20 @@ export async function action({ request }: ActionFunctionArgs) {
   const session = await sessionService.getSession(request);
   session.set('newHomeAddress', parsedDataResult.data);
 
-  // TODO the validateWSAddress(..) call below must be corrected to consume the correct response
-  // const wsAddressWebService = await getWSAddressService();
-  // const { address, city, country, postalCode, province } = parsedDataResult.data;
-  // const validateResultAddress = await wsAddressWebService.validateWSAddress({ address, city, country, postalCode: postalCode ?? '', province: province ?? '' });
-
-  // TODO the following line must be removed once the validateWSAddress(..) call above is corrected
-  const validateResultAddress = { statusCode: 'Valid' };
-
-  const redirectUrl = validateResultAddress.statusCode === 'Invalid' ? '/personal-information/home-address/address-accuracy' : '/personal-information/home-address/confirm';
+  const wsAddressService = await getWSAddressService();
+  const { address, city, country, postalCode, province } = parsedDataResult.data;
+  const correctedAddress = await wsAddressService.correctAddress({ address, city, country, postalCode: postalCode ?? '', province: province ?? '' });
```

Comment:
I don't like the default empty strings here. Can `wsAddressService` be modified to make these parameters optional?

Reply:
Made parameters optional [here](https://github.com/DTS-STN/canadian-dental-care-plan/compare/80533e5d722577969bcd8f923d88caab329d3bae..6d58543589c408a034c61f8a0209748698a0befc#diff-a330c9ede4049228d6825394285f16e4e4737fb401eb33bdfaa0021111a6bb85).